ozaria/site/views/play/level/tome/SpellView.coffee

```diff
@@ -833,49 +840,73 @@ module.exports = class SpellView extends CocoView
   # * The editor will reserve space for one annotation as a codeless area.
   # - Problem alerts and ranges will only show on fully cast worlds. Annotations will show continually.
 
+  fetchToken: (source, language) =>
+    if language not in ['java', 'cpp']
+      return Promise.resolve(source)
+    else if source of @loadedToken
+      return Promise.resolve(@loadedToken[source])
+
+    headers =  { 'Accept': 'application/json', 'Content-Type': 'application/json' }
+    m = document.cookie.match(/JWT=([a-zA-Z0-9.]+)/)
```

Comment:
are we using this variable?

Reply:
interesting. seems we don't. let me double check it and clean it